### PR TITLE
chore: have a static GitHub star button

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright.dev/issues/474

### CODE REVIEW

File: src/components/GitHubStarButton/LICENSE [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (pavelfeldman) [inline src/components/GitHubStarButton/LICENSE:191]: Why are we adding Apache 2 here and who holds the rights?

File: src/components/GitHubStarButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './index.css';
+
+type StarButtonProps = {
+    owner: string
+    repo: string
+}
+
+const STARS = '35k+';
```
Comment (pavelfeldman) [inline src/components/GitHubStarButton/index.tsx:9]: We are 34

**Review by pavelfeldman — APPROVED** [review]

File: src/components/GitHubStarButton/index.tsx (hunk 2 above)
Comment (mxschmitt) [inline src/components/GitHubStarButton/index.tsx:9]:
```suggestion
const STARS = '34k+';
```

File: src/components/GitHubStarButton/LICENSE (hunk 1 above)
Comment (mxschmitt) [inline src/components/GitHubStarButton/LICENSE:191]:
```suggestion
   Copyright 2011 Mark Otto
```

**Pull request merged**